README.md

```diff
@@ -1 +1,115 @@
-# review-project
\ No newline at end of file
+# review-project
+
```

Comment:
- `required`
- **Well done so far👏👏**
- [ ] Please ensure adding and highlighting what the student did well with this project so far 👏🏼 , You can list the highlights in the code review body above 👍🏼 

- [ ] I suggest you add emoji's to your comments so that the student understands the tone you are trying to communicate 😎.

- [ ] I also strongly suggest that in order to sound polite you need to be using words like `please`, `kindly`, `I suggest`, and `I highly recommend` ...etc, and many other words you find properly.

- [ ] Well done with explaining **why** you are asking for that changes, however, it would be great 👌 if you provide some examples 👨‍🏫 and it will be even greater if you share some resources ✔️  so the student can learn something 😄 as well and don't feel the change was unfair 👌.